tools/cli/commands/build.js

```diff
@@ -831,10 +857,33 @@ async function buildProject( t ) {
 		await fs.writeFile( `${ buildDir }/.gitattributes`, rules, { encoding: 'utf8' } );
 	}
 
+	// Get the proeject version number from the changelog.md file.
+	let projectVersionNumber = '';
+	try {
+		const changelogText = await fs.readFile( `${ t.cwd }/CHANGELOG.md`, { encoding: 'utf8' } );
+		projectVersionNumber = changelogText
+			.toString()
+			.split( '\n' )
+			.find( line => {
+				return line.startsWith( '## ' ) && ! line.includes( '-alpha' );
```

Comment:
The `! line.includes( '-alpha' )` part here seems incorrect. When a changelog looks like this
```
## 2.0.0-alpha
...

## 1.2.3
...
```
we don't want to claim the version is 1.2.3! We want 2.0.0.

Reply:
> We want 2.0.0.

I may be a bit confused here. My intent was to grab the topmost non-alpha version. In your example `2.0.0-alpha` hasn't been published yet. Don't we want to only grab the most recent published package versions for example?

Reply:
We want the version of the package from the monorepo. Whether the dep comes out as "2.0.0" or "2.0.0-alpha" or "2.0.0@dev", I don't know which might be better.

Reply:
Is it safe to assume that we should always use the topmost changelog entry then?

Reply:
b69bfcc